internal/authtoken/authtoken.go

```diff
@@ -86,25 +89,31 @@ const (
 )
 
 func newAuthTokenId() (string, error) {
+	const op = "authtoken.newAuthTokenId"
 	id, err := db.NewPublicId(AuthTokenPrefix)
 	if err != nil {
-		return "", fmt.Errorf("new auth token id: %w", err)
+		//WIP Note: It looks like this bubbles up the errs made in `newId` so I don't think we need `WithCode` here.
+		return "", errors.Wrap(err, op)
 	}
 	return id, err
 }
 
 // newAuthToken generates a token with a version prefix.
 func newAuthToken() (string, error) {
+	const op = "authtoken.newAuthToken"
 	token, err := base62.Random(tokenLength)
 	if err != nil {
-		return "", fmt.Errorf("unable to generate auth token: %w", err)
+		//WIP Note: Is there a better option than Unknown to use here? Looks like
+		// the errors bubbled up from UUID are either nilerr or unable to read
+		return "", errors.Wrap(err, op, errors.WithCode(errors.Unknown))
```

Comment:
We should ideally never return an `Unknown` code, perhaps we could use `errors.Io` or make a new `Rand` code.  When doing this please fix the same issue here that I missed in a previous refactor https://github.com/hashicorp/boundary/blob/main/internal/db/id.go#L26 oops

As a side note there is no need to provide `Unknown` explicitly here as it will default to `Unknown` anyway